tests/testthat/test-drive_mkdir.R

```diff
@@ -56,52 +56,57 @@ test_that("drive_mkdir() creates a folder in root folder", {
 test_that("drive_mkdir() accepts parent folder given as dribble", {
   skip_on_travis()
   skip_on_appveyor()
+  on.exit(drive_delete(nm_("a")))
```

Comment:
This file was leaving lots of un-cleaned up files behind on each test run (my fault!). Between fixing this and eliminating similar behaviour in tests for `drive_cp()` (not my fault!), I confirm that a successful test run does not change the number of files owned by our test account. I have also now deleted the >200 files we had accumulated this way. 😧